packages/backend/migration/1637320813000-forwarded-report.js

```diff
@@ -0,0 +1,13 @@
+const { QueryRunner } = require('typeorm');
+
+module.exports = class forwardedReport1637320813000 {
+	name = 'forwardedReport1637320813000';
```

Comment:
Will this migration apply correctly if there is another migration with a higher number? `1639325650583-chart-v3.js` was merged yesterday